data/src/test/java/org/apache/iceberg/data/DataTestHelpers.java

```diff
@@ -57,7 +56,7 @@ public static void assertEquals(Types.ListType list, List<?> expected, List<?> a
   public static void assertEquals(Types.MapType map, Map<?, ?> expected, Map<?, ?> actual) {
     Type valueType = map.valueType();
 
-    Assert.assertEquals("Map size should match", expected.size(), actual.size());
+    assertThat(actual).as("Map size should match").hasSize(expected.size());
```

Comment:
same as mentioned above on how to properly compare sizes between collections

Reply:
done